### PR TITLE
Add a description field for SEO love

### PR DESCRIPTION
**Pull request by richard-still-ft**

### CODE REVIEW

File: server/controllers/article.js [hunk 1]
```diff
@@ -99,6 +99,14 @@ module.exports = function articleV3Controller (req, res, next, content) {
 	decorateMetadataHelper(content);
 	content.isSpecialReport = content.primaryTag && content.primaryTag.taxonomy === 'specialReports';
 
+	// Setup the description field
+	content.description = '';
+	if (content.summaries) {
```
Comment (matthew-andrews) [inline server/controllers/article.js]: i'd suggest swapping this logic around because `summaries` is deprecated

**Comment by matthew-andrews** [comment]
it's good, suggested a small tweak

+1

File: server/controllers/article.js (hunk 1 above)
Comment (richard-still-ft) [inline server/controllers/article.js]: right o

**Comment by matthew-andrews** [comment]
💃

**Pull request merged**